app/pb_kits/playbook/pb_button_toolbar/_button_toolbar.scss

```diff
@@ -73,37 +109,164 @@
       }
   
       &[class*=_horizontal] {
-        & > [class^=pb_button]  {
-          &:not(:last-child){
-            border-bottom-right-radius: 0;
-            border-top-right-radius: 0;
-            border-right-width: 0;
-            border: 1px solid #e1e4e8;
+        &[class*=_primary] {
+          & > [class^=pb_button]:not(:first-child)::before {
+            @include vertical_border($opacity: $opacity_1, $border_color: true)
+          }
+          & > [class^=pb_button]:not(:first-child):hover::before {
+            @include vertical_border($opacity: 0, $border_color: false)
           }
-          &:not(:first-child){
-            border-bottom-left-radius: 0;
-            border-top-left-radius: 0;
-            border-left-width: 0;
-            border: 1px solid #e1e4e8;
+          [class^=pb_button]:hover + [class^=pb_button]::before {
+            @include vertical_border($opacity: 0, $border_color: false)
+          }
+          & > [class^=pb_button]  {
+            &:not(:last-child){
+              border-bottom-right-radius: 0;
+              border-top-right-radius: 0;
+              border-right-width: 0;
+            }
+            &:not(:first-child){
+              border-bottom-left-radius: 0;
+              border-top-left-radius: 0;
+              border-left-width: 0;
+            }
+          }
```

Comment:
This seems to be the same as lines 145-155 can you eliminate one by moving it to the parent class?